code/game/objects/items/tools/wirebrush.dm

```diff
@@ -0,0 +1,13 @@
+/obj/item/wirebrush
```

Comment:
Looks to me like you did more than just refactor code! Put it in the changelog and in your PR.